src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/UdtTest/SqlServerTypesTest.cs

```diff
@@ -251,6 +251,7 @@ private static void TestUdtSqlDataReaderGetStream(CommandBehavior behavior)
             }
         }
 
+#if NETCOREAPP
```

Comment:
Please use the `ActiveIssue` attribute and include more info in the related issue.